lab1/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Goblin Mine</title>
+	<link rel="stylesheet" href="app.css">
+
+</head>
+<body>
+	<h1>Goblin Mine</h1>
+
+	<div>
+		<p class="story">
+			Click button to mine ore <br>
+			Goblins rush to your mine seeking work <br>
+			Your ore is regarded as the best that can be mined <br>
+		</p>
```

Comment:
Suggestion – it's probably better to have each line as a `<p>` tag if you want a line break in between each line.